modules/ROOT/pages/exchange1-faq.adoc

```diff
@@ -42,7 +42,7 @@ image::ex1-category-objects.png[Object categories settings buttons]
 +
 . Click Save Changes after editing terms, or Discard to quit any changes you made.
 
-Note: When you create a category, you must be in the Master Organization. If you create a category in a business group, the category does not appear.
+Note: When you create a category, you must be in the root organization. If you create a category in a business group, the category does not appear.
```

Comment:
I think we either use the NOTE notice format or incorporate it into the sentence:
```suggestion
Note that when you create a category, you must be in the root organization. If you create a category in a business group, the category does not appear.
```
Does not appear where?